index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SciFacts homepage</title>
+    <link rel="stylesheet" href="./css/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,300&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+    <nav>
+    <div id="topbar">
+        <ul class="icons">
+            <li>
+                <a href="www.twiiter.com/mike-dev"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li> 
+                <a href="www.linkedin/mike-dev"><i class="fa fa-linkedin"></i></a>
+            </li>
+        </ul> 
+        <a href="#"><p>English</p></a>
+        <a href="#"><p>My page</p></a>
+    </div>
+    </nav>
+
+    <nav id="navbar">
+        <a href="index.html"><div id="navbar_logo"></div></a>
+        <ul id="nav_liststyle">
+            <li><a href="index.html" class="menu_list home_color">Home</a></li>
+            <li><a href="about.html" class="menu_list">About</a></li>
+        </ul>
+    </nav>
+
+    <header id="header">
+        <div id="large_screen">
+            <div class="hero_content">
+                <nav id="menu_icon">
+                    <a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a>
+                </nav> 
+                <p class="hello_text">"Hello! great learners"</p>
+                <h1>Science Fun Facts</h1>
+                <p class="brief">The history of science is almost as old as time itself. Science is constantly working 
+                    from the edges of the universe to the innermost cells that make you function
+                </p>
+                <p class="date">All about science</p>
+                <p class="location">Its Latin word, <i>scienta</i> means knowledge.</p>
+            </div> 
```

Comment:
- [ ] Kindly  Increase the margin from ther left so as to make your design look similar to the given design
- [ ] Also increase the font size for the h1 tag so as to make it look more similar to the given design

#### Given design
![image](https://user-images.githubusercontent.com/81932478/190408203-e63a6681-e57c-4f2d-9d0e-80ecb0e6e167.png)

#### Your design
![image](https://user-images.githubusercontent.com/81932478/190408337-314a368f-7b27-4cad-8d61-d49f66020fab.png)